MIG-1504: MTC 1.8.2 release notes (#67775)
openshift/openshift-docs: modules/migration-mtc-release-notes-1-8.adoc

In this release, after enabling `Require SSL verification` and adding content to the CA bundle file for an MCG NooBaa bucket in MigStorage, the connection fails as expected. However, when reverting these changes by removing the CA bundle content and clearing `Require SSL verification`, the connection still fails. The issue is only resolved by deleting and re-adding the repository. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2240052[*BZ#2240052*])

.Backup phase fails after setting custom CA replication repository

> In {mtc-full} ({mtc-short}), after editing the replication repository, and adding a custom CA, and the repo is successfully connected, when triggering a migration, there is a failure returned on the backup phase.

Reviewer: ```suggestion
In {mtc-full} ({mtc-short}), after a user edited the replication repository, added a custom CA certificate, successfully connected the repository, and triggered a migration, a failure occurred during the backup phase.
```
- See previous comment on similar issue.

> This issue has been resolved in in {mtc-short} 1.8.2.

Reviewer: ```suggestion
This issue has been resolved in {mtc-short} 1.8.2.
```
- Remove repeated word
- I think you can delete this line as redundant. You've already mentioned that these are fixed issues, correct?

.CVE-2023-26136: tough-cookie package before 4.1.3 are vulnerable to Prototype Pollution

> In {mtc-short}, versions of the package tough-cookie before 4.1.3, used in {mtc-short}, are vulnerable to Prototype Pollution because of the improper handling of Cookies when using CookieJar in `rejectPublicSuffixes=false` mode.
> 
> It is advised to update to {mtc-short} 1.8.2 or later, which resolve this issue.

Reviewer: ```suggestion
In previous releases of ({mtc-short}), versions prior to 4.13 of the `tough-cookie` package used in {mtc-short} were vulnerable to prototype pollution. This vulnerability occurred because CookieJar did not handle cookies properly when the value of the `rejectPublicSuffixes` setting was set to `false` [maybe also mention in what config file??].

To resolve the issue, update to {mtc-short} 1.8.2 or later.
```
I have the same comments as in the previous nearly identical item. Please revise per those suggestions.

For more details, see link:https://access.redhat.com/security/cve/cve-2023-26136[(CVE-2023-26136)]

.CVE-2022-25883 openshift-migration-ui-container: nodejs-semver: Regular expression denial of service

> In previous releases of ({mtc-short}), versions of the package semver before 7.5.2, used in {mtc-short}, are vulnerable to Regular Expression Denial of Service (ReDoS) from the function new Range, when untrusted user data is provided as a range.
> 
> It is advised to update to {mtc-short} 1.8.2 or later, which resolve this issue.

Reviewer: ```suggestion
In previous releases of ({mtc-short}), versions prior to 7.5.2 of the `semver` package used in {mtc-short} were vulnerable to Regular Expression Denial of Service (ReDoS) attacks when untrusted user data was provided as a range in the new `Range` function.

To resolve the issue, update to {mtc-short} 1.8.2 or later.
```

I have the same comments and suggested edits as in the previous nearly identical item.

For more details, see link:https://access.redhat.com/security/cve/cve-2022-25883[(CVE-2022-25883)]

> 

Reviewer 2: [These](https://github.com/openshift/openshift-docs/pull/67775/files#diff-f1f0c55ea489db1d283dfe452b491e98f1c8179beb042cb810fe81f3a2f648f9R49-R75) are under 1.8 "known issues".  I would expect them to be found under 1.8.2 "resolved issues" [section](https://github.com/openshift/openshift-docs/pull/67775/files#diff-f2ddeabe0025f31b3a1aef70b1da720407483c0ae248594cb5980a5ace1fb9e9R8-R14).

Author: sorry forgot to save the 1.8.2 release notes doc
